src/utils/getPriceGuidance.ts

```diff
@@ -33,7 +37,7 @@ export const getPriceGuidance = async (slug: string) => {
     let avgPrice
     let hasNoBasePrice
     const collectionPricesInDollars = flatMap(
-      results.marketingCollection.artworks.hits,
+      results.marketingCollection.artworks.artworks_connection.edges,
       artwork => {
```

Comment:
Related to previous comment, I would have thought that what you get here is not an artwork but a node:  `{ node: { priceCents: 42} }`, no?